articles/quickstart/native/xamarin/index.yml

```diff
@@ -25,10 +25,10 @@ github:
   org: auth0-samples
   repo: auth0-xamarin-oidc-samples
 requirements:
-  - Visual Studio 2017 or Visual Studio for Mac
-  - Xamarin for Visual Studio 4.5
+  - Visual Studio 2017, 2019 or Visual Studio for Mac
```

Comment:
Do you think we'd be better off saying something like "Visual Studio 2017 or later" here? What happens when VS 2021 comes out?

Reply:
`Visual Studio 2017 or later or Visual Studio for Mac` reads a bit weird I think. What do u think about `Visual Studio 2017+ or Visual Studio for Mac`

Reply:
That works 👍